hack/e2e-internal/e2e-up.sh

```diff
@@ -29,16 +29,40 @@ source "${KUBE_ROOT}/cluster/kube-util.sh"
 
 prepare-e2e
 
+function calc_next_cidr {
```

Comment:
You can't go to 100K pods with /16 because that only gives us 65K addresses. I don't like dumbing down things just because it looks simple now esp. when we have a better solution that works.

Also, it is not as simple as `CURRENT_NET_NUMBER+=1`. You probably could do that with /16 and

```shell
CURRENT_NET_NUMBER=180

function calc_next_cidr {
    CURRENT_CIDR="10.${CURRENT_NET_NUMBER}.0.0/16"
    ...
}
```

But it gets a little more trickier with /14 addresses. The solution in this PR is the right solution to this problem and it works. We shouldn't try to dumb it down.